app/services/order_details/param_updater.rb

```diff
@@ -84,12 +86,17 @@ def update_attributes(params)
   end
 
   def trigger_notifications
-    OrderDetails::DisputeResolvedNotifier.new(@order_detail,current_user: @editing_user).notify
+    OrderDetails::DisputeResolvedNotifier.new(@order_detail, current_user: @editing_user).notify
     OrderDetails::AssignmentNotifier.new(@order_detail).notify
   end
 
   private
 
+  def log_and_rollback
+    logger.error("errors for reservation #{@order_detail.reservation.id}: #{@order_detail.reservation.errors.full_messages}")
```

Comment:
Would it be worth using the ActiveSupport notifications? That should trigger a notification to rollbar, which would actively inform us rather than us thinking to check the logs.

```ruby
ActiveSupport::Notifications.instrument("background_error", exception: e, information: "Errors for reservation #{@order_detail.reservation.id}: #{@order_detail.reservation.errors.full_messages}")
```